src/webview/helm-chart/app/fromHelm.tsx

```diff
@@ -6,15 +6,9 @@ import * as React from 'react';
 import 'react-dom';
 import { HelmSearch } from './helmSearch';
 
-type FromHelmProps = {
-    titleText: string
-};
-
-export function FromHelm(props: FromHelmProps) {
+export function FromHelm() {
```

Comment:
This component just wraps `HelmSearch`. I think you can delete this component, and use `<HelmSearch/>` instead of `<FromHelm/>` in `src/view/helm-chart/app/home.tsx`